apps/node_monitoring/src/monitors/delegate.rs

```diff
@@ -438,6 +460,13 @@ impl DelegatesMonitor {
             }
         }
 
+        let path = format!("/dev/shell/automaton/action_stats_for_blocks");
```

Comment:
Shouldn't we query stats for near blocks here, like above?

Reply:
Yeah true, before we were only maintaining last 6 blocks, so I just simplified by storing all stats. Will change that.

Reply:
but nvm, actually there is a limit for rpc. on how many block stats can be retrieved. Limit is set to 64.
https://github.com/tezedge/tezedge/blob/2779c7e43988e51e28133c3c0b8fd81f822a0eea/shell_automaton/src/rpc/rpc_effects.rs#L145

So this will fetch last 64 blocks and store them. In report, more data might come in handy. @akoptelov what do you think?